examples/django_example/django_example/tests/test_api.py

```diff
@@ -2,7 +2,6 @@
 from django.test import TestCase
 from django.urls import reverse
 
-
```

Comment:
Two blank lines before a top level function is preferred to adhere to pep8:

https://peps.python.org/pep-0008/#blank-lines